### ISSUE TITLE
Bitfile path on target is hard to work with.

### ISSUE THREAD

**Issue by theSloopJohnB** [LabVIEW-DCAF/TagEditorCore#321]
I've been having some trouble with the 'bitfile path on target relative to root' field that's on the cRIO configuration.

It says 'relative to /home/lvuser' but when I put the file name in the error message indicates that it tries to load it from the root directory.

Here's a snippet to illustrate the problem:

![image](https://user-images.githubusercontent.com/26978265/26896891-1227f52e-4b8c-11e7-9d36-92ed74a2a4d6.png)

I also can't really edit the bitfile path on target field because LabVIEW 'helpfully' converts my path to windows style. Maybe we should use a string here.

**Comment by pollockm** [comment]
Yeah, the utter worthlessness of the path control when working on a cross-target application is a sore spot with us.  We've done some work on other modules (TDMS included) to swap in a string or a path control depending on which target is being used.  There is also some ugly string parsing work put in previously to "de-correct" the path.  @becega might be able to comment as to where that lived.
